### DISCUSSION TITLE
Update/upgrade procedure?

### DISCUSSION THREAD

**Discussion by Alex00101** [OrcaSlicer/OrcaSlicer#1676] [General]
Was trying to install 1.6.4 beta and seems like all my printers and sett8ngs are gone (I have backups but that would restore the whole 1.6.3). I'm trying to manually copy all my json files to the new 1.6.4 folder but no matter what I do, I have any Default printer.

Appreciate any help

**Comment by SoftFever** [comment]
Hi Alex,

They will come back once you re-start the application

**Reply by Alex00101** [reply]
Thanks, my issue was actually that I had my ow set of configs (json) for my printer, which I had to revert back top generic Klipper one to keep 1.6.4 working. All's good after I've done that.
